langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -82,6 +111,10 @@ public isolated function substring(string str, int startIndex, int endIndex = st
 # but the ordering will often not be consistent with cultural expectations
 # for sorted order.
 #
+# ```ballerina
+# int codePointCompare = "Austria".codePointCompare("Australia");
```

Comment:
Shall we wrap "Austria" to a variable and then use. It will more clear.

Reply:
I don't feel any extra clearance after moving it to a variable. But +1 for this to maintain the consistency

Reply:
+1